lib/workers/repository/updates/generate.ts

```diff
@@ -73,13 +73,17 @@ export function generateBranchConfig(
   const depNames: string[] = [];
   const newValue: string[] = [];
   const toVersions: string[] = [];
+  const toValues: Set<string> = new Set();
```

Comment:
```suggestion
  const toValues = new Set<string>();
```
A little bit simpler.